onthequad/onthequad/DataBaseCalls.py

```diff
@@ -8,14 +8,14 @@ class DataBaseCalls:
 
```

Comment:
could you please also change the name of this file to EventDataBase or something to avoid confusion later? I named my UsersDB.

Reply:
Changed to to be EventDataBase